packages/ti_recordedfuture/data_stream/threat/manifest.yml

```
@@ -106,49 +63,41 @@ streams:
         show_user: true
       - name: interval
         type: text
-        title: Interval between risklist downloads.
+        title: Interval between risklist downloads
         description: Use Go Duration syntax (eg. 1h)
         default: "1h"
         multi: false
         required: true
         show_user: true
       - name: timeout
         type: text
-        title: Request timeout for the risklist download.
+        title: Request timeout for the risklist download
         description: >
           Must provide enough time for downloading and processing the risklist. Valid time units are ns, us, ms, s, m, h.
 
         default: "5m"
         multi: false
         required: true
         show_user: true
-      - name: api_token
-        type: password
-        title: API Token
-        description: Recorded Future API Token (RF_TOKEN).
-        multi: false
+      - name: preserve_original_event
         required: true
         show_user: true
-        secret: true
-      - name: custom_url
-        type: url
-        title: Custom URL
-        description: URL to download a custom Fusion File.
+        title: Preserve original event
+        description: Preserves a raw copy of the original event, added to the field `event.original`
+        type: bool
         multi: false
-        required: false
-        show_user: false
-      - name: endpoint
-        type: url
-        title: API Endpoint
-        description: Base API URL.
+        default: false
+      - name: enable_request_tracer
+        type: bool
+        title: Enable request tracing
         multi: false
-        required: true
+        required: false
         show_user: false
-        default: https://api.recordedfuture.com/v2
-      - name: proxy_url
+        description: The request tracer logs requests and responses to the agent's local file-system for debugging configurations. Enabling this request tracing compromises security and should only be used for debugging. See [documentation](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-httpjson.html#_request_tracer_filename) for details.
```

```suggestion
        description: The request tracer logs requests and responses to the agent's local file-system for debugging configurations. Enabling this request tracing compromises security and should only be used for debugging. See [documentation](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-cel#_resource_tracer_enable) for details.
```

Updating to CEL input option instead of httpjson.